tests/e2e/submit-enketo-form.wdio-spec.js

```diff
@@ -89,37 +90,29 @@ describe('Submit Enketo form', () => {
     },
   };
 
-  beforeAll(async () => {
+  before(async () => {
+    await loginPage.cookieLogin();
     await utils.seedTestData(userContactDoc, docs);
   });
 
-  afterEach(async () => {
-    await utils.afterEach();
-  });
-
   it('submits on reports tab', async () => {
-    await commonElements.goToReportsNative();
-
-    await helper.waitElementToBeClickable(reportsPo.submitReport);
+    await commonElements.goToReports();
+    await (await reportsPo.submitReportButton()).waitForClickable();
 
     // select form
-    await helper.clickElementNative(reportsPo.submitReport);
-    await helper.waitElementToBeClickable(reportsPo.firstForm);
-    await helper.clickElementNative(reportsPo.firstForm);
+    await (await reportsPo.submitReportButton()).click();
+    await (await reportsPo.firstForm()).click();
```

Comment:
I think It would be less problematic if we would reuse `reportsPo.openForm` instead of clicking on the first form, in case some other forms will appear in the list.